optimism/src/keccak/column.rs

```diff
@@ -313,7 +338,7 @@ impl<G: Send + std::fmt::Debug> FromParallelIterator<G> for KeccakColumns<G> {
         I: IntoParallelIterator<Item = G>,
     {
         let mut iter_contents = par_iter.into_par_iter().collect::<Vec<_>>();
-        let next = iter_contents
+        let next: [G; ZKVM_KECCAK_COLS_NEXT] = iter_contents
```

Comment:
I would suppose the typing annotation is required, therefore you added it?

Reply:
Oh it was probably not needed and I just forgot it was there.